modules/olm-installing-from-operatorhub-using-cli.adoc

```diff
@@ -314,6 +325,41 @@ spec:
 <1> Include the client ID.
 <2> Include the tenant ID.
 <3> Include the subscription ID.
+====
+
+* If the cluster is in {gcp-wid-short} mode, include the following fields:
++
+.Example `Subscription` object with {gcp-wid-short} variables
+[%collapsible]
+====
+[source,yaml]
+----
+kind: Subscription
+# ...
+spec:
+ config:
+   env:
+   - name: AUDIENCE
+     value: "<audience_url>" <1>
+   - name: SERVICE_ACCOUNT_EMAIL
+     value: "<service_account_email>" <2>
+----
+
+where:
+
+`<audience>`:: Created in {gcp-short} by the administrator when they set up {gcp-wid-short}, the `AUDIENCE` value must be a preformatted URL in the following format:
++
+[source,text]
+----
+//iam.googlespis.com/projects/<project_number>/locations/global/workloadIdentityPools/<pool_id>/providers/<provider_id>
+----
+`<service_account_email>`:: The `SERVICE_ACCOUNT_EMAIL` value is a {gcp-short} service account email that is impersonated during Operator operation, for example:
++
+[source,text]
+----
+<service_account_name>@<project_id>.iam.gserviceaccount.com
+----
```

Comment:
Oops meant to take the callouts here out... 

And in general FYI: I would rather have stuck with callouts to match the previous substeps, but codeblocks in callout text do not render a scrollbar, so the really-long "preformatted URL" (which I wanted to duplicate here convenience) would break the page width. So I've switched just this one substep to the "where:" style. Not ideal, but hopefully not too jarring.